test/integration/query-interface.test.js

```diff
@@ -41,6 +41,28 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     });
   });
 
+  describe('showAllTables', () => {
+    it('should contain only tables', function() {
```

Comment:
Please change this to `should not contain views`, I think it is more descriptive

```suggestion
    it('should not contain views', function() {
```